components/Header/src/index.scss

```diff
@@ -0,0 +1,10 @@
+.denhaag-header {
+  border-bottom: var(--denhaag-header-border-bottom);
+
+  &__container {
```

Comment:
Seems odd to call the content of a container (the header in this case) container. Could we maybe call it `__content`?